resources/install.rb

```diff
@@ -0,0 +1,160 @@
+property :sql_reboot, kind_of: [TrueClass, FalseClass], default: true
+property :security_mode, kind_of: String, equal_to: ['Windows Authenication', 'Mixed Mode Authentication'], default: 'Windows Authenication'
+property :sa_password, kind_of: String
+property :sysadmins, kind_of: [Array, String], required: true, default: ['Administrator']
```

Comment:
required with a default doesn't really do much except prevent passing a nil there which is already prevented by the fact that this is a string. You can remove the required bit.